actors/builtin/market/market_test.go

```diff
@@ -933,23 +1311,44 @@ func (h *marketActorTestHarness) getDealState(rt *mock.Runtime, dealID abi.DealI
 	return s
 }
 
-func (h *marketActorTestHarness) terminateDeals(rt *mock.Runtime, dealIDs []abi.DealID, minerAddr address.Address) {
+func (h *marketActorTestHarness) assertDealsTerminated(rt *mock.Runtime, epoch abi.ChainEpoch, dealIds ...abi.DealID) {
+	for _, d := range dealIds {
+		s := h.getDealState(rt, d)
+		require.EqualValues(h.t, epoch, s.SlashEpoch)
+	}
+}
+
+func (h *marketActorTestHarness) assertDeaslNotTerminated(rt *mock.Runtime, dealIds ...abi.DealID) {
+	for _, d := range dealIds {
+		s := h.getDealState(rt, d)
+		require.EqualValues(h.t, market.EpochUndefined, s.SlashEpoch)
```

Comment:
If this was the only reason to export EpochUndefined, just code the `-1` here instead and leave it hidden.

Reply:
This is done.